### ISSUE TITLE
Learning Bound Lipschitz Covering Number Bound

### ISSUE THREAD

**Issue by lakshmiDRIP** [lakshmiDRIP/DROP#409]
_LipschitzCoveringNumberBound_ contains the Upper Bounds of the Covering Numbers induced by Lipschitz and approximate Lipschitz Loss Function Class. The References are:

 * Bartlett, P. L., P. Long, and R. C. Williamson (1996): Fat-shattering and the Learnability of Real-Valued Functions <i>Journal of Computational System Science</i> <b>52 (3)</b> 434-452

 * Anthony, M., and P. L. Bartlett (1999): <i>Artificial Neural Network Learning - Theoretical Foundations</i> <b>Cambridge University Press</b> Cambridge, UK
